rust/gui-client/src-tauri/firezone-client-ipc.service

```diff
@@ -7,6 +7,8 @@ CapabilityBoundingSet=CAP_CHOWN CAP_NET_ADMIN
 DeviceAllow=/dev/net/tun
 LockPersonality=true
 LogsDirectory=dev.firezone.client
+# Allow users in `firezone` group to delete log files
+LogsDirectoryMode=775
```

Comment:
This would allow users in `firezone` write bogus log files. I think the threat from that is small, and the only alternative I had in mind is issuing a "ClearLogs" command to the IPC service and waiting for a response, which would add a lot of complexity to both the IPC protocol and the log clearing code.

So we could do that later if we really need.

But since the GUI decides what goes into the zip file in the end, it may not even matter if it can modify the original logs.

Reply:
Yeah, if we re-enable log uploading.

Depending how tight we want the security sandbox, I was thinking the tunnel process shouldn't be allowed to read the user's files, so it would be safer to let the user read the tunnel's files.